### ISSUE TITLE
Vagrant failing to install arachni

### ISSUE THREAD

**Issue by tjcwilk** [gauntlt/gauntlt-starter-kit#10]
on vagrant up im getting the following errors.

==> default: STDERR: ERROR:  Error installing arachni:
==> default:    ERROR: Failed to build gem native extension.
==> default: 
==> default:         /usr/bin/ruby1.9.1 extconf.rb
==> default: checking for libkern/OSAtomic.h... no
==> default: creating Makefile
==> default: 
==> default: make
==> default: compiling atomic_boolean.c
==> default: In file included from atomic_boolean.c:5:0:
==> default: ruby_193_compatible.h: In function ‘rb_error_arity’:
==> default: ruby_193_compatible.h:20:3: warning: passing argument 2 of ‘rb_raise’ makes pointer from integer without a cast [enabled by default]
==> default:    rb_raise(rb_eTypeError, err_mess);
==> default:    ^
==> default: In file included from /usr/include/ruby-1.9.1/ruby.h:32:0,
==> default:                  from atomic_boolean.c:1:
==> default: /usr/include/ruby-1.9.1/ruby/ruby.h:1172:27: note: expected ‘const char _’ but argument is of type ‘VALUE’
==> default:  PRINTF_ARGS(NORETURN(void rb_raise(VALUE, const char_, ...)), 2, 3);
==> default:                            ^
==> default: /usr/include/ruby-1.9.1/ruby/ruby.h:42:3: note: in definition of macro ‘PRINTF_ARGS’
==> default:    decl **attribute**((format(printf, string_index, first_to_check)))
==> default:    ^
==> default: /usr/include/ruby-1.9.1/ruby/ruby.h:1172:13: note: in expansion of macro ‘NORETURN’
==> default:  PRINTF_ARGS(NORETURN(void rb_raise(VALUE, const char_, ...)), 2, 3);
==> default:              ^
==> default: In file included from atomic_boolean.c:5:0:
==> default: ruby_193_compatible.h:20:3: error: format not a string literal and no format arguments [-Werror=format-security]
==> default:    rb_raise(rb_eTypeError, err_mess);
==> default:    ^
==> default: cc1: some warnings being treated as errors
==> default: make: *_\* [atomic_boolean.o] Error 1
==> default: 
==> default: 
==> default: Gem files will remain installed in /var/lib/gems/1.9.1/gems/concurrent-ruby-ext-1.0.0 for inspection.
==> default: Results logged to /var/lib/gems/1.9.1/gems/concurrent-ruby-ext-1.0.0/ext/concurrent/gem_make.out
==> default: ---- End output of /usr/bin/ruby -S gem install gauntlt bundler arachni ----
==> default: Ran /usr/bin/ruby -S gem install gauntlt bundler arachni returned 1
==> default: [2016-03-22T14:19:59+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
Chef never successfully completed! Any errors should be visible in the
output above. Please fix your recipes so that they properly complete.

**Comment by apolloclark** [comment]
Yup. We've recently made a bunch of updates to Gauntlt, adding support up to Ruby 2.3, which will fix this. Once we've locked in a release, I'll update this vagrant box as well.
